StreamChatUITestsAppUITests/Tests/PushNotification_Tests.swift

```diff
@@ -33,11 +33,9 @@ final class PushNotification_Tests: StreamTestCase {
     
     func test_goToBackgroundFromChannelList_and_tapOnPushNotification() throws {
         linkToScenario(withId: 291)
-        
-        throw XCTSkip("[CIS-2164] The test app is not yet ready for this test")
 
         GIVEN("user goes to channel list") {
-            userRobot.login()
+            userRobot.login().openChannel().tapOnBackButton()
```

Comment:
Why is this needed here? 🤔

Reply:
To make the mock server know which channel the user has entered to send push notifications to that channel.

Reply:
Maybe we can add a comment explaining this? It would be nice

Reply:
Done 👍